src/Components/ConfirmationModal.tsx

```diff
@@ -26,17 +29,38 @@ interface ComponentToggleButtonProps {
     | 'outline-info'
     | 'outline-dark'
     | 'outline-light';
+    secondaryVariant?: | 'primary'
```

Comment:
Is that first `|` supposed to be there? Also, this is kinda long, maybe we should make a type for it?

Reply:
it's copypasta from the original `confirmVariant`

Reply:
updated